src/Cli.Tests/ConfigureOptionsTests.cs

```diff
@@ -246,6 +246,165 @@ public void TestUpdateMultipleParametersForGraphQLSettings()
             Assert.AreEqual(updatedAllowIntrospectionValue, runtimeConfig.Runtime.GraphQL.AllowIntrospection);
         }
 
+        /// <summary>
+        /// Tests that running "dab configure --runtime.rest.enabled" on a config with various values results
+        /// in runtime. Takes in updated value for rest.enabled and 
+        /// validates whether the runtime config reflects those updated values
+        [DataTestMethod]
+        [DataRow(false, DisplayName = "Update enabled to be false for Rest.")]
+        [DataRow(true, DisplayName = "Update enabled to be true for Rest.")]
+        public void TestUpdateEnabledForRestSettings(bool updatedEnabledValue)
+        {
+            // Arrange -> all the setup which includes creating options.
+            SetupFileSystemWithInitialConfig(INITIAL_CONFIG);
+
+            // Act: Attempts to update enabled flag
+            ConfigureOptions options = new(
+                runtimeRestEnabled: updatedEnabledValue,
+                config: TEST_RUNTIME_CONFIG_FILE
+            );
+            Assert.IsTrue(TryConfigureSettings(options, _runtimeConfigLoader!, _fileSystem!));
+
+            // Assert: Validate the Enabled Flag is updated
+            string updatedConfig = _fileSystem!.File.ReadAllText(TEST_RUNTIME_CONFIG_FILE);
```

Comment:
what is the change here?

Reply:
The ask is not to have asserts in your Act.
Act involves executing the code that you want to test. It is the part where you invoke the method under test with specific inputs.
you're using Assert(TryConfigureSettings). 

you can seperate it out, get the value in the Act and do the validation in the Assert section.

This will help in clarity and maintainability.